geode-web/src/test/java/org/apache/geode/management/internal/cli/commands/ConnectCommandWithSSLTest.java

```diff
@@ -265,7 +271,26 @@ public void connectWithHttpSSL() throws Exception {
         "security-properties-file", sslConfigFile.getAbsolutePath());
     assertThat(gfsh.isConnected()).isFalse();
 
-    // can conect to http
+    // can connect to http
+    gfsh.connect(locator.getHttpPort(), GfshShellConnectionRule.PortType.http,
+        "security-properties-file", sslConfigFile.getAbsolutePath(), "skip-ssl-validation", "true");
+    assertThat(gfsh.isConnected()).isTrue();
+  }
+
+  @Test
+  public void connectWithHttpSSLAndSkipSSLValidation() throws Exception {
+    httpSslPropertiesSkipValidation.store(out, null);
```

Comment:
so this test shows that we do not even need either keystore or truststore to connect? I thought we at least need a keystore....

Reply:
From the client perspective, a keystore is only needed if we're doing mutual auth, i.e. the client needs a certificate to present to the server.